backend/src/main.rs

```diff
@@ -40,9 +40,9 @@ async fn main() -> std::io::Result<()> {
             .wrap(middleware::Compress::default())
             .wrap(middleware::Logger::default())
             .service(web::resource("/ws/").route(web::get().to(echo_ws)))
+            .service(web::scope("/api").route("/game/{id:\\d+}/play", web::post().to(api::game::play::game_play)))
```

Comment:
write integration test for this

Reply:
tbh I don't know who to write integration tests for actix yet and the whole thing is a bit unstable atm, so I am going to postpone it

Reply:
it has a ticket now